### PR TITLE
PEXDDPS-2032 Update cookie banner with new layout

### PR DESCRIPTION
**Pull request by DJerzewski**

### CODE REVIEW

File: components/hmpo-cookie-banner/_style.scss [hunk 1]
```diff
@@ -1,13 +1,31 @@
...
+
+.hmpo-cookie-banner__text {
+    margin: 0 0 govuk-spacing(2);
+}
+
+.hmpo-cookie-banner__button {
+    @include mq($until: desktop) {
+        width: 100%;
+        margin: 0 0 govuk-spacing(4) 0;
+    }
```
Comment (HughePaul) [inline components/hmpo-cookie-banner/_style.scss:23]: have the mobile/tablet (mobile first) as the default css for this class, i.e. not inside a media query,
then have the desktop and bigger in a media query.

File: components/hmpo-template.njk [hunk 2]
```diff
@@ -24,8 +24,13 @@
...
         {% from "hmpo-cookie-banner/macro.njk" import hmpoCookieBanner %}
-        {{ hmpoCookieBanner({
-            html: translate("govuk.cookieBanner")
+        {{ hmpoCookieBanner(ctx, {
+            title: translate("govuk.cookieBanner.title"),
+            html: translate("govuk.cookieBanner.content") ,
+            accept: translate("govuk.cookieBanner.accept"),
+            preferences: translate("govuk.cookieBanner.preferences"),
+            acceptHref: hmpoAcceptCookies,
+            preferencesHref: hmpoCookiePreferences
```
Comment (HughePaul) [inline components/hmpo-template.njk:33]: It's tempting to put this component in an `{% if hmpoAceptCookies and hmpoCookiePreferences %}` so its not rendered by default on pages that don't specify the urls.

(the components are used on other sites, most of them will kill the cookieBanner block if they don't want it)

(perhaps show the old component if the urls aren't specified?)

Comment (HughePaul) [reply components/hmpo-template.njk:33]: Alternatively, perhaps the component can be explicitly used by shared lib, and the cookies etc aren't set here, but by middleware, and it shown by shared lib?

It made sense for it here when it was really generic message, but now its a bit more custom to DCS

Comment (DJerzewski) [reply components/hmpo-template.njk:33]: That is a good suggestion - I have moved the banner to shared lib instead.

**Comment by DJerzewski** [comment]
Closing as the functionality moved to shared lib instead

**Pull request closed without merging**